src/screens/EditWorkout/EditWorkoutPage.tsx

```diff
@@ -1,21 +1,86 @@
 import { NativeStackScreenProps } from '@react-navigation/native-stack';
-import React from 'react';
-import { View, Text, TouchableOpacity } from 'react-native';
+import React, { useState } from 'react';
+import { FlatList, KeyboardAvoidingView, Platform, StyleSheet } from 'react-native';
+import AddButton from '../../components/AddButton';
 import { NavigatorParamList } from '../DrawerNavigator';
+import { Exercise, PLACEHOLDER_EXERCISE_NAME } from '../../types/Exercise';
+import ExerciseBlock from '../../components/ExerciseBlock';
+import { updateExerciseInWorkout, useEditableWorkout } from '../../hooks/useEditableWorkout';
+import useSelectedWorkout from '../../hooks/useSelectedWorkout';
 
 type EditWorkoutPageProps = NativeStackScreenProps<NavigatorParamList, 'EditWorkoutPage'>;
 
 const EditWorkoutPage: React.FC<EditWorkoutPageProps> = ({ navigation }) => {
-  const navigateToEditExercisePage = () => {
+  const [selectedWorkout, setSelectedExercise] = useSelectedWorkout((state) => [
```

Comment:
const [selectedWorkout, **setSelectedWorkouts**] ?

Reply:
`selectedWorkout` is to display the workout, `setSelectedExercise` is for setting the exercise to view when we navigate to the edit exercise page.